tests/system/test_session_api.py

```diff
@@ -699,6 +711,162 @@ def _transaction_read_then_raise(transaction):
     assert rows == []
 
 
+@pytest.mark.skipif(
+    not _helpers.USE_EMULATOR,
+    reason="Emulator needed to run this tests",
+)
+@pytest.mark.skipif(
+    not ot_helpers.HAS_OPENTELEMETRY_INSTALLED,
+    reason="Tracing requires OpenTelemetry",
+)
+def test_transaction_abort_then_retry_spans(sessions_database, ot_exporter):
```

Comment:
Looks like there is lot of redundant code here that is there in test_observability_options.py.

Can we move all these new tests to test_observability_options.py? It will help to have one single place for new observability tests?